packages/protocol/contracts/common/interfaces/ICeloDistributionSchedule.sol

```diff
@@ -5,7 +5,7 @@ interface ICeloDistributionSchedule {
   /**
    * @notice Sets the distribution schedule dependencies during L2 transition.
    */
-  function activate(uint256, uint256, address, uint256, address) external;
+  function activate(uint256, uint256, address, uint256) external;
```

Comment:
I didn't know that an interface could have more parameters than the implementation and still work, nice catch